sqlmesh/core/model/definition.py

```diff
@@ -1949,6 +1951,11 @@ def _create_model(
     model_schema_name = exp.to_table(name, dialect=dialect).db
     physical_schema_override: t.Optional[str] = None
 
+    if not defaults:
+        # Set query optimization to "True" by default, unless specified otherwise by
+        # the user in the Model flags
+        kwargs["optimize"] = kwargs.get("optimize", True)
```

Comment:
why was this added here? is this the proper way to do config defaults? why aren't there other places that do this pattern?

Reply:
It doesn't look like it's needed yeah, will remove